autobuild/templates/dockercompose.j2

```diff
@@ -205,6 +236,9 @@ services:
 {% endif %}
 {% if deploy_avax %}
       - avax
+{% endif %}
+{% if deploy_nevm %}
+      - sys
```

Comment:
Darn, I missed this before, but caught it now that I'm trying to build the payment processor locally and getting an error.
```
- sys
``` 
Needs to be all caps, like this:
```
- SYS
```

Reply:
Thanks, I spotted that yesterday, haven't pushed the fix yet. Hopefully tomorrow.